gallery/tutorials/plot_apple_picker.py

```diff
@@ -0,0 +1,63 @@
+"""
+Apple Picker
+============
+
+We demonstrate ASPIRE's particle picking methods using the ``Apple`` class.
+"""
+
+import matplotlib.pyplot as plt
+import mrcfile
+
+from aspire.apple.apple import Apple
+
+# %%
+# Load Micrograph Data
+# --------------------
+#
+# Initiate ASPIRE's ``Apple`` class and load the micrograph data.
+
+apple_picker = Apple()
+filename = "data/falcon_2012_06_12-14_33_35_0.mrc"
+
+# %%
+# Pick Particles and Find Centers
+# -------------------------------
+#
+# Here we use the ``process_micrograph`` method from the ``Apple`` class to find particles in the micrograph.
+
+centers = apple_picker.process_micrograph(filename, show_progress=False)
+
+# %%
+# Read Micrograph
+# ---------------
+
+with mrcfile.open(filename, mode="r") as mrc:
+    micro_img = mrc.data
+
+# %%
+# Verify Dimensions
+# -----------------
+#
+# Here we peek at the image dimensions and observe the number of particles picked.
+
+micro_img.shape
```

Comment:
I was thinking that maybe for size sanity checks we can do an assert instead of printing.  This documents the sanity check and totally skips the whole printing business.

assert something.attribute == expected_size

Reply:
There are three attributes being checked in this section of code: 

micro_img.shape 
apple_picker.tau2
centers.shape 

I'm not sure there are particular values expected here. For instance, centers.shape returns an nX2 array, where n is the number of particles picked. This appears to be more of an inspection of that value, which leads me to think that this value should actually be displayed.

Reply:
Updated using logger for now.